Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/MenuFileActivityIntegrationTest.java

```diff
@@ -203,7 +199,7 @@ public void testLoadImageDialogOnBackPressed() {
 		onDrawingSurfaceView()
 				.check(matches(isDisplayed()));
 	}
-
+/* Enable if help button is enabled again
```

Comment:
Please remove code instead of commenting out. We can restore it from VCS at a later time if needed.

Reply:
Instead, please add a test verifying that the option is gone.